src/systems/filecoin_vm/indices/indices.go

```diff
@@ -62,70 +62,78 @@ func (inds *Indices_I) StorageDeal_ClientCollateralBounds(
 	panic("")
 }
 
-func (inds *Indices_I) BlockReward_SectorPower(
+func (inds *Indices_I) BlockReward_SectorWeight(
 	sectorSize sector.SectorSize,
 	startEpoch block.ChainEpoch,
 	endEpoch block.ChainEpoch,
 	dealWeight deal.DealWeight,
-) block.StoragePower {
+) block.SectorWeight {
 	// for every sector, given its size, start, end, and deals within the sector
 	// assign sector power for the duration of its lifetime
 	PARAM_FINISH()
 	panic("")
 }
 
 func (inds *Indices_I) BlockReward_PledgeCollateralReq(
-	minerActiveStoragePower block.StoragePower,
-	minerInactiveStoragePower block.StoragePower,
+	minerActiveSectorWeight block.SectorWeight,
+	minerInactiveSectorWeight block.SectorWeight,
 	minerPledgeCollateral actor.TokenAmount,
 ) actor.TokenAmount {
 	PARAM_FINISH()
 	panic("")
 }
 
-func (inds *Indices_I) BlockReward_StoragePowerProportion(minerActiveStoragePower block.StoragePower) util.BigInt {
-	// return proportion of StoragePower for miner
+func (inds *Indices_I) BlockReward_SectorWeightProportion(minerActiveSectorWeight block.SectorWeight) util.BigInt {
+	// return proportion of SectorWeight for miner
 	PARAM_FINISH()
 	panic("")
 }
 
-func (inds *Indices_I) BlockReward_CollateralPowerProportion(minerPledgeCollateral actor.TokenAmount) util.BigInt {
+func (inds *Indices_I) BlockReward_PledgeCollateralProportion(minerPledgeCollateral actor.TokenAmount) util.BigInt {
 	// return proportion of Pledge Collateral for miner
 	PARAM_FINISH()
 	panic("")
 }
 
-func (inds *Indices_I) BlockReward_EarningPowerProportion(
-	minerActiveStoragePower block.StoragePower,
+func (inds *Indices_I) StoragePower(
+	minerActiveSectorWeight block.SectorWeight,
+	minerInactiveSectorWeight block.SectorWeight,
 	minerPledgeCollateral actor.TokenAmount,
+) block.StoragePower {
+	// return StoragePower based on inputs
+	// StoragePower for miner = func(ActiveSectorWeight for miner, PledgeCollateral for miner, global indices)
+	PARAM_FINISH()
+	panic("")
```

Comment:
I think we should add our best guess for now

ie if collat req is met, return weight from active sectors